node/network/bitfield-distribution/src/tests.rs

```diff
@@ -41,11 +41,12 @@ use sp_keystore::{testing::MemoryKeystore, Keystore, KeystorePtr};
 
 use std::{iter::FromIterator as _, sync::Arc, time::Duration};
 
+const TIMEOUT: Duration = Duration::from_millis(300);
```

Comment:
Isn't that a bit too much ? could hide other issues ? I was thinking something la 50ms would be fine.

Reply:
It is universally used in other tests despite of the incorrect description, that's why I have decided to use the same constant here instead of adding another one. But there is one more place where 10ms is used. I'll update to use 50ms timeout there as well.